pkg/sql/sqlbase/structured.go

```diff
@@ -3091,6 +3156,82 @@ func (desc *MutableTableDescriptor) MakeMutationComplete(m DescriptorMutation) e
 					return err
 				}
 			}
+		case *DescriptorMutation_ComputedColumnSwap:
+			args := t.ComputedColumnSwap
+
+			oldCol, err := desc.FindColumnByID(args.OldColumnId)
+			if err != nil {
+				return err
+			}
+			newCol, err := desc.FindColumnByID(args.NewColumnId)
+			if err != nil {
+				return err
+			}
+
+			newCol.Hidden = false
```

Comment:
I dont think you want to be touching hidden here. Hidden is like for the rowid implicit column.